### PR TITLE
Adds straight_road_with_crosswalk_integration.xodr map.

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Related to #46 

## Summary
 - Adds straight_road_with_crosswalk_integration.xodr map.

image

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

**Review by liangfok — CHANGES_REQUESTED** [review]

File: standalone/straight_road_with_crosswalk_integration/straight_road_with_crosswalk_integration.xodr [hunk 1]
```diff
@@ -0,0 +1,758 @@
...
+                        </userData>
+                    </lane>
+                </right>
+            </laneSection>
+        </lanes>
+    </road>
+    <road name="Road 11" length="4.0" id="11" junction="10">
+        <link>
+            <predecessor elementType="road" elementId="9" contactPoint="end"/>
+            <successor elementType="road" elementId="11" contactPoint="start"/>
```
[the remark this answers is not in the document]
Comment (francocipollone) [reply standalone/straight_road_with_crosswalk_integration/straight_road_with_crosswalk_integration.xodr:579]: Oh, that's a bug indeed, thanks for pointing that out, I will fix it.

[the remark this answers is not in the document]
Comment (liangfok) [reply standalone/straight_road_with_crosswalk_integration/straight_road_with_crosswalk_integration.xodr:579]: In our downstream use cases, we confirmed that after changing `elementId="11"` to `elementId="10"`, the map passed our tests.

[the remark this answers is not in the document]
Comment (francocipollone) [reply standalone/straight_road_with_crosswalk_integration/straight_road_with_crosswalk_integration.xodr]: Done [bb059ac](https://github.com/maliput/maliput_xodr/pull/58/commits/bb059ac127787e1622fcf18c6b49b767b48e93b2)

**Comment by francocipollone** [comment]
> In our downstream use cases, we confirmed that after changing elementId="11" to elementId="10", the map passed our tests.

@liangfok Can we go ahead and merge this then?

**Review by liangfok — APPROVED** [review]
LGTM Thanks!

**Pull request merged**